Core/Goals/PullTargetGoal.cs

```diff
@@ -39,7 +39,8 @@ public PullTargetGoal(ILogger logger, ConfigurableInput input, PlayerReader play
             AddPrecondition(GoapKey.incombat, false);
             AddPrecondition(GoapKey.hastarget, true);
             AddPrecondition(GoapKey.pulled, false);
-            AddPrecondition(GoapKey.withinpullrange, true);
+            AddPrecondition(GoapKey.targetisalive, true);
```

Comment:
Why have to check if the target is alive? What's have to do with the this goal? 

To my best knowledge in `FollowRouteGal` searches for the possible target, but it does not acquire corpses